modules/fbx/editor/editor_scene_importer_ufbx.cpp

```diff
@@ -48,8 +48,9 @@ void EditorSceneFormatImporterUFBX::get_extensions(List<String> *r_extensions) c
 Node *EditorSceneFormatImporterUFBX::import_scene(const String &p_path, uint32_t p_flags,
 		const HashMap<StringName, Variant> &p_options,
 		List<String> *r_missing_deps, Error *r_err) {
-	if (p_options.has("fbx/importer") && int(p_options["fbx/importer"]) == FBX_IMPORTER_FBX2GLTF) {
-		Ref<EditorSceneFormatImporterFBX> fbx2gltf_importer;
+	// FIXME: Hack to work around GH-86309.
+	if (p_options.has("fbx/importer") && int(p_options["fbx/importer"]) == FBX_IMPORTER_FBX2GLTF && GLOBAL_GET("filesystem/import/fbx2gltf/enabled")) {
```

Comment:
Added a check for whether the fbx2gltf importer is actually enabled. Needs testing.